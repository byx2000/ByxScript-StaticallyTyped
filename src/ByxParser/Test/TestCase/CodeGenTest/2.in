function void main()
{
	int i = 10;
	int j;
	double x = 3.14;
	double y;
	int k = i;
	double z = y;
	j = 300;
	j = k;
	y = 1.5;
	z = x;
	
	int v1;
	double height = 0.124;
	v1 = height;
	height = v1;
}