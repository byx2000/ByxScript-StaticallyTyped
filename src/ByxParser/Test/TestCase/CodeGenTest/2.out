1
8 0

0

iconst 10
istore 0
iconst 0
istore 1
dconst 3.14
dstore 2
dconst 0
dstore 3
iload 0
istore 4
dload 3
dstore 5
iconst 300
istore 1
iload 4
istore 1
dconst 1.5
dstore 3
dload 2
dstore 5
iconst 0
istore 6
dconst 0.124
dstore 7
dload 7
istore 6
iload 6
dstore 7
ret